src/main/scala/is/hail/io/vcf/ExportVCF.scala

```diff
@@ -44,10 +44,8 @@ object ExportVCF {
       case TString(_) =>
         sb.append(TString.loadString(m, offset))
       case TCall(_) =>
-        val p = Genotype.gtPair(m.loadInt(offset))
-        sb.append(p.j)
-        sb += '/'
-        sb.append(p.k)
+        val c = m.loadInt(offset)
+        sb.append(Call.toString(c))
```

Comment:
This is pretty common, so adding a Call.addtoStringBuffer(sb) might be reasonable here.